### PR TITLE
Launchpad: Remove unscoped css style from Celebrate Launch modal

### PR DESCRIPTION
**Pull request by sharpirate**
This is a follow up to this PR https://github.com/Automattic/wp-calypso/pull/75275#discussion_r1160660752

### Time Estimate
Review: Short
Test: Short

### Summary
This PR removes unscoped styles from `celebrate-launch.scss` to the tooltip component's own stylesheet to avoid unexpected style bleed.

### Testing
1. Checkout this branch
2. Create a new Launchpad site and proceed to launch it
3. Once you get redirected to My Home you will see the celebrate launch modal appear
4. Click the Copy site button and confirm you still see the tooltip **Copied to clipboard!**

image

### CODE REVIEW

**Comment by jeyip** [comment]
@nuriapenya Do you happen to know who else we'd need to talk to about making a change to the shared tooltip component z index style?

**Comment by jeyip** [comment]
From the technical side of things,  things look fine. We'll want to smoke test other instances of tooltips though

File: client/components/tooltip/style.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 $large-tooltip-box-shadow-color: rgba(0, 0, 0, 0.05);
 
 .tooltip.popover {
+
+	// always display the popover on top even inside a modal with a high z index
+	z-index: 100000;
```
Comment (simison) [inline client/components/tooltip/style.scss]: I remember we used to have some sass functions to manage z-indexes ([similar to Gutenberg](https://github.com/WordPress/gutenberg/blob/308f3ee5ce02192d6129e0764d62ecf5c5ebb6e1/packages/components/src/tooltip/style.scss#L2)).

It makes it easier to manage different z-indexes in different components collectively.

Can you look up if that exists and use it here? Thanks!

**Comment by simison** [comment]
Looks good — I haven't tested but I'd test modals, dropdowns, "lightboxes" or "[scrims](https://m2.material.io/design/environment/surfaces.html#attributes)" to ensure it works as expected; above modals and dropdowns but below scrims for example.

I left technical nitpick which can help manage above.

Also asked Team Calypso to review but I don't think that has to be blocking merging if you're confident this works. Up to you!

File: client/assets/stylesheets/shared/functions/_z-index.scss [hunk 2]
```diff
@@ -172,6 +172,7 @@ $z-layers: (
 
 		".is-section-woocommerce .global-notices": 999999,
 		".webpack-build-monitor": 99999999,
+		".tooltip.popover": 100000,
```
Comment (simison) [inline client/assets/stylesheets/shared/functions/_z-index.scss]: I think you'd put this somewhere higher on the list, they seem to be in numerical order?

Comment (sharpirate) [reply client/assets/stylesheets/shared/functions/_z-index.scss]: Thanks, updated!

**Comment by sharpirate** [comment]
> Looks good — I haven't tested but I'd test modals, dropdowns, "lightboxes" or "[scrims](https://m2.material.io/design/environment/surfaces.html#attributes)" to ensure it works as expected; above modals and dropdowns but below scrims for example.

Hey @simison, do you know how I can find instances of all of those to test with the popup? Not sure if there are specific classnames corresponding to each one

**Comment by simison** [comment]
I'd probably look from code with component name, or create dummy page with all sorts of components mixed to ensure it works all together. Playing with devdocs might also help, since all components should be there in one page https://wpcalypso.wordpress.com/devdocs — they're just not combined together, e.g. tooltip in a modal etc.

**Comment by sharpirate** [comment]
I couldn't find any obvious issues when testing, so I'll go ahead and deploy this. If anything pops up I can address it in a follow up

**Pull request merged**